### PR TITLE
Release changes from main

### PR DESCRIPTION
**Pull request by ygree**
Release changes from main

### CODE REVIEW

**Comment by ignasi35** [comment]
maybe wait for @thothmoses in case there's some incosistency?

File: docs-source/docs/modules/deployment/pages/gcp-sql.adoc [hunk 1]
```diff
@@ -4,24 +4,24 @@
 include::partial$include.adoc[]
 
 https://cloud.google.com/products/databases[GCP Cloud SQL {tab-icon}, window="tab"] provides managed SQL databases of several database engines.
-This describes how to use PostgreSQL because that is what is used in the xref:microservices-tutorial:index.adoc[Implementing Microservices with Akka tutorial] but it would be similar for the other database engines.
+This describes how to use PostgreSQL because that is what is used in the xref:microservices-tutorial:index.adoc[Implementing Microservices with Akka tutorial, but it would be similar for the other database engines.
```
Comment (ignasi35) [inline docs-source/docs/modules/deployment/pages/gcp-sql.adoc:7]: just noticed thre's a syntax error in the link here (no closing bracket)

Comment (ygree) [reply docs-source/docs/modules/deployment/pages/gcp-sql.adoc]: Good catch, Ignasi!

**Review by thothmoses — APPROVED** [review]

**Comment by thothmoses** [comment]
@ygree  looks good to merge.

**Pull request merged**